createuser.php

```diff
@@ -1,5 +1,51 @@
 <?php
-require_once('php/connection.php');
-function validateUser($email, $password) {
-    
-}
\ No newline at end of file
+
+session_start();
+if(isset($_SESSION["auth"])){  
+
+if($_SESSION["auth"] == "2")  
+{
+ 
+}
+else
+{
+    header("Location: login.php?e=4");
+ die("You are not Signed in.");  }  
+}
+
+$sql = "UPDATE
+`users`
+SET
+`firstname` = '$f_name',
+`lastname` = '$l_name',
+`email` = '$email',
+`password` = '$password'
+WHERE
+`users`.`userid` = $uid";
+
+if(mysqli_query($db_connect,$sql))
```

Comment:
This will not work.
Look to your `auth.php` file as an example of what your code should look like, but it will need changing.